cfn-resources/access-list-api-key/cmd/resource/resource.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	atlasSDK "go.mongodb.org/atlas-sdk/v20231115001/admin"
+	atlasSDK "go.mongodb.org/atlas-sdk/v20231115002/admin"
```

Comment:
is this PR created by an action?

Reply:
yes, the one I fixed

Reply:
please don't merge yet, I'm going to do some tests